src/utils/format-numbers.js

```diff
@@ -53,14 +53,28 @@ export function formatPercent(value: number): string {
 }
 
 export function formatBytes(bytes: number): string {
-  if (bytes < 4 * 1024) {
+  if (bytes < 10000) {
+    // Use singles up to 10,000.  I think 9,360B looks nicer than 9.36KB.
```

Comment:
hummm I'm really not sure about that, but because we were already doing it for numbers less than 4K I guess there's no strong reason to not do it until 9999.

Reply:
Yeah, I chose the original 4k arbitrarilly.  At least 10k matches the formatSI implementation, maybe that means it makes more sense?  I dono. ;-)